File: day1/main.go
package main

import (
	"fmt"
)

func promptUser() {
	var noun string
	var verb string
	var adj string

	fmt.Print("Provide a noun: ")
	fmt.Scan(&noun)

	fmt.Print("Provide a verb: ")
	fmt.Scan(&verb)

	fmt.Print("Provide a adj: ")
	fmt.Scan(&adj)

	fmt.Printf("The %s %s likes to %s the cloud server.\n", adj, noun, verb)
}

func main() {
	fmt.Println("I need your help telling a story.")
	promptUser()
}

File: day1/main.py
def prompt_user() -> str:
    """Ask the user for a noun, verb, and adjective."""
    noun = input("Provide a noun: ")
    verb = input("Provide a verb: ")
    adj = input("Provide a adj: ")

    return print(f"The {adj} {noun} likes to {verb} the cloud server.")

def main():
    """execute main program"""
    print("I need your help telling a story.")
    prompt_user()


if __name__ == "__main__":
    main()
    